plugins/woocommerce/includes/class-wc-frontend-scripts.php

```diff
@@ -120,7 +120,7 @@ private static function get_asset_url( $path ) {
 	 * @param  string   $version   String specifying script version number, if it has one, which is added to the URL as a query string for cache busting purposes. If version is set to false, a version number is automatically added equal to current installed WordPress version. If set to null, no version is added.
 	 * @param  boolean  $in_footer Whether to enqueue the script before </body> instead of in the <head>. Default 'false'.
 	 */
-	private static function register_script( $handle, $path, $deps = array( 'jquery' ), $version = WC_VERSION, $in_footer = true ) {
+	private static function register_script( $handle, $path, $deps = array( 'jquery' ), $version = WC_VERSION, $in_footer = array( 'strategy' => 'defer' ) ) {
```

Comment:
Instead of this being an `$in_footer` arg, it seems it should be renamed to `$args`, yeah?

Reply:
that would make it a little clearer and a better match for core.